drizzlepac/haputils/generate_custom_svm_param_file.py

```diff
@@ -0,0 +1,242 @@
+        'use_defaults_configs' parameter is ignored. The default is None.
+
+    output_custom_pars_file: string, optional
+        Fully specified output filename which contains all the configuration parameters
+        available during the processing session.  The default is None.
+
+    phot_mode : str, optional
+        Which algorithm should be used to generate the sourcelists? 'aperture' for aperture photometry;
+        'segment' for segment map photometry; 'both' for both 'segment' and 'aperture'.
+        Default value is 'both'.
+
+    log_level : int, optional
+        The desired level of verboseness in the log statements displayed on the screen and written to the
+        .log file. Default value is 20, or 'info'.
+
+
+    RETURNS
+    -------
+    return_value: integer
+        A return exit code used by the calling Condor/OWL workflow code: 0 (zero) for success, 1 for error
+    """
+    # This routine needs to return an exit code, return_value, for use by the calling
+    # Condor/OWL workflow code: 0 (zero) for success, 1 for error condition
+    return_value = 0
+    log.setLevel(log_level)
+    if not clobber:
+        if os.path.exists(output_custom_pars_file):
+            msg = "A file named '{}' already exists. Please choose a unique name for the custom SVM parameter file.".format(output_custom_pars_file)
+            log.critical(msg)
+            sys.exit()
+    # Define trailer file (log file) that will contain the log entries for all processing
+    if isinstance(input_filename, str):  # input file is a poller file -- easy case
+        logname = input_filename.replace('.out', '.log')
+    else:
+        logname = 'svm_param_gen.log'
+    # Initialize total trailer filename as temp logname
+    logging.basicConfig(filename=logname, format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
+    # start processing
+    starting_dt = datetime.datetime.now()
+    log.info("Run start time: {}".format(str(starting_dt)))
+
+    # Start by reading in any environment variable related to catalog generation that has been set
+    # cat_switches = {sw: _get_envvar_switch(sw, default=envvar_cat_svm[sw]) for sw in envvar_cat_svm}
+
+    total_obj_list = []
+    product_list = []
+    try:
+        # Parse the poller file and generate the the obs_info_dict, as well as the total detection
+        # product lists which contain the ExposureProduct, FilterProduct, and TotalProduct objects
+        # A poller file contains visit data for a single instrument.  The TotalProduct discriminant
+        # is the detector.  A TotalProduct object is comprised of FilterProducts and ExposureProducts
+        # where its FilterProduct is distinguished by the filter in use, and the ExposureProduct
+        # is the atomic exposure data.
+        log.info("Parse the poller and determine what exposures need to be combined into separate products.\n")
+        obs_info_dict, total_obj_list = poller_utils.interpret_obset_input(input_filename, log_level)
```

Comment:
The consequence of this operation is that the input files will be copied and renamed, something which the user may not be expecting unless it is added to the documentation somewhere.

Reply:
Issue resolved by commit [8caec28](https://github.com/spacetelescope/drizzlepac/pull/896/commits/8caec281b0dc569c2894b945372480fb1c926471)